sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -243,18 +242,17 @@ - (void)dealloc {
   [self.player destroy];
 }
 
-@end
-
-@implementation OOSkinViewController(Internal)
-
-- (void)toggleFullscreen {
+- (void)setFullscreen:(BOOL)fullscreen {
+  if (fullscreen == _fullscreen) return;
```

Comment:
I could see this happening if the method was called several times in a row. Something weird may happen in that case since we're pausing and resuming the player a lot of times, that may make it out of sync.